src/Make_cyg_ming.mak

```diff
@@ -106,6 +106,13 @@ else
 TERMINAL=no
 endif
 
+# Set to yes to enable sound support.
+ifeq (HUGE, $(FEATURES))
```

Comment:
`ifneq ($(findstring $(FEATURES),BIG HUGE),)` might be better?
Sound is supported with the big features on Linux.